pkg/k8scontext/context.go

```diff
@@ -10,7 +10,7 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
Hm... 😕

Reply:
remove this ?

Reply:
Oh no no - I was just scratching my head wondering where else `mapset` came if not from here -- I see it being used in the code below... essentially - how did this work to begin with :)
But now looking at the deckarp source - it is `package mapset` so that's how that worked.
What you did is great - explicit!